session.go

```diff
@@ -21,32 +21,37 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/coreos/go-oidc/jose"
 	"go.uber.org/zap"
+	"gopkg.in/square/go-jose.v2/jwt"
 )
 
 // getIdentity retrieves the user identity from a request, either from a session cookie or a bearer token
-func (r *oauthProxy) getIdentity(req *http.Request) (*userContext, error) {
+func (r *oauthProxy) getIdentity(req *http.Request) (*userContext, string, error) {
```

Comment:
Seems like `rawToken` is already sort of returned through `userContext`. I feel that it is neater not to have this function do more than promised by also returning the `rawToken` as one of the return values.

Reply:
fixed